.github/workflows/script.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python3
+# -*- coding: UTF-8 -*-
```

Comment:
Change the name of the script to be more self-explanatory. Do this for the other PR too